scripts/env

```diff
@@ -104,7 +104,11 @@ env_list() {
 env_diff() {
 	env_init
 	env_sync_data
-	git diff --cached --color
+	if [ -t 1 ]; then
+		git diff --cached --color
+	else
+		git diff --cached
```

Comment:
Then it would be required to append to the "no terminal" part '--color=newer'.

Or what might even work is to replace the original line with

    git diff --cached --color=auto

and remove the manual stdout FD check

Reply:
Amended.